foxpuppet/browser/navbar.py

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class Navbar(object):
+    def __init__(self, selenium, *args, **kwargs):
+        self.selenium = selenium
+        self._locationbar = None
+        self._menu_button = (By.ID, 'PanelUI-menu-button')
+
+    def open_window(self, private=False):
+        self.selenium.set_context('chrome')
+        if private:
+            self.selenium.find_element(*self._menu_button).click()
```

Comment:
Yes, the reason also is that this path is always available. The menu button could have been customized away and reachable via another location. And shortcuts are locale related, so we miss the l10n support. The menu path is also the default for puppeteer.